yaksh/templates/yaksh/question.html

```diff
@@ -161,33 +161,11 @@
                  </div>
                  <div class="panel-body">
                      {% if question.type == "mcq" %}
```

Comment:
For MCQ and MCC, error message is shown only when user do not select any option and submits.
Error message is shown, "Please select a valid option"